pages/room/index.js

```diff
@@ -18,8 +19,13 @@ const CreateRoom = () => {
     initSession();
   }, []);
 
-  function create() {
+  async function create() {
     const id = uuid();
+    const result = await axios.post('/api/create-new-room', {
+      name: 'test name',
+      description: 'test descritpion',
+      roomId: id,
```

Comment:
I see you pass the roomId here but it doesn't look like you're saving that to your roomSchema. Make sure you do that later.